portfolio/src/main/webapp/index.html

```diff
@@ -25,84 +26,177 @@
-        <label for="new-task-estimated-length">Estimated length:</label>
-        <input type="text" id="new-task-estimated-length" name="new-task-estimated-length" />
-        <select id="new-task-estimated-length-unit" name="new-task-estimated-length-unit">
-          <option value="minute">minutes(s)</option>
-          <option value="hour">hour(s)</option>
-          <option value="day">day(s)</option>
-        </select>
-        <br />
-        <label for="new-task-priority">Task's priority:</label>
-        <select id="new-task-priority" name="new-task-priority">
-          <option value="5">5 (High)</option>
-          <option value="4">4</option>
-          <option value="3">3</option>
-          <option value="2">2</option>
-          <option value="1">1 (Low)</option> </select
-        ><br />
-        <button type="button" class="btn btn-primary">Add this task</button>
+    <nav class="navbar navbar-expand-lg navbar-light bg-light">
+      <a class="navbar-brand" href="#">Task Scheduler</a>
+      <div class="collapse navbar-collapse" id="navbarSupportedContent">
+        <ul class="navbar-nav mr-auto">
+          <li class="nav-item dropdown">
+            <a
+              class="nav-link dropdown-toggle"
+              href="#"
+              id="navbarDropdown"
+              role="button"
+              data-toggle="dropdown"
+              aria-haspopup="true"
+              aria-expanded="false"
+            >
+              Scheduling Preferences
+            </a>
+            <div class="dropdown-menu" aria-labelledby="navbarDropdown">
+              <a>Working Hours:</a>
+              <form class="px-4 py-3">
+                <input
+                  type="time"
+                  id="working-hour-start"
+                  name="working-hour-start"
+                  class="form-control"
+                  value="09:00"
+                />
+                <a>to</a>
+                <input type="time" id="working-hour-end" name="working-hour-end" class="form-control" value="17:00" />
+              </form>
+            </div>
+          </li>
+        </ul>
       </div>
-    </div>
+    </nav>
+    <div class="row">
+      <div id="new-task-wrapper" class="col-md-12 col-lg-6 col-xl-3">
+        <h2>Add the tasks you would like to schedule for today</h2>
+        <form id="new-task-form">
+          <div class="form-group">
+            <label for="new-task-name">Task name:</label>
+            <input type="text" id="new-task-name" class="form-control" name="new-task-name" placeholder="New Task" />
+          </div>
 
-    <div id="new-task-list-wrapper">
-      <h2>Tasks added:</h2>
-    </div>
+          <label for="new-task-description">Task description:</label><br />
+          <textarea
+            id="new-task-description"
+            class="form-control"
+            name="new-task-description"
+            placeholder="Description"
+          ></textarea
```

Comment:
This doesn't look right. Ptal at the style guide section below.
https://google.github.io/styleguide/htmlcssguide.html#HTML_Line-Wrapping

Reply:
After running the linter, I am left with
`
<textarea
            id="new-task-description"
            class="form-control"
            name="new-task-description"
            placeholder="Description"
          >
          </textarea>
          <br />
`

Reply:
`<textarea`
`            id="new-task-description"`
`            class="form-control"`
`            name="new-task-description"`
`            placeholder="Description"`
`          >`
`          </textarea>`
`          <br />`

Reply:
After running the linter I am left with what is above, however within the textarea tag's content, only 2 indents are used as opposed to the four indicated by the style guide. The style guide also says to put the textarea tag closing bracket on the same line as `placeholder="Description"` but the linter moves it down a line.